docs/containers-policy.json.5.md

```diff
@@ -70,6 +70,11 @@ a host/namespace/image stream.
 *Note:* The _hostname_ and _port_ refer to the Docker registry host and port (the one used
 e.g. for `docker pull`), _not_ to the OpenShift API host and port.
 
+More general scopes are prefixes of individual-image scopes, and specify a repository (by omitting the tag or digest),
+a repository namespace, or a registry host (by only specifying the host name)
```

Comment:
- The `*note*` paragraph should probably be after, not in the middle of, the scope discussion.
- Hum, the more detailed scopes are “image streams”, not “repositories” — and already described above; so, maybe just continue the “host namespace/image stream” sentence with “, or a wildcarded…”.

Reply:
done